src/test/javascript/spec/component/overview/course-statistics/course-statistics.component.spec.ts

```diff
@@ -238,20 +233,41 @@ describe('CourseStatisticsComponent', () => {
         fixture.detectChanges();
         comp.ngOnInit();
         fixture.detectChanges();
-        expect(comp.groupedExercises.length).to.equal(1);
+        expect(comp.groupedExercises.length).toEqual(1);
         const modelingWrapper = fixture.debugElement.query(By.css('#modeling-wrapper'));
-        expect(modelingWrapper.query(By.css('h2')).nativeElement.textContent).to.exist;
-        expect(modelingWrapper.query(By.css('#absolute-score')).nativeElement.textContent).to.exist;
-        expect(modelingWrapper.query(By.css('#reachable-score')).nativeElement.textContent).to.exist;
-        expect(modelingWrapper.query(By.css('#max-score')).nativeElement.textContent).to.exist;
-        expect(fixture.debugElement.query(By.css('#presentation-score')).nativeElement.textContent).to.exist;
+        expect(modelingWrapper.query(By.css('h2')).nativeElement.textContent).not.toBe(null);
```

Comment:
`not.toBe(null)` sounds wrong here, I think you can use something like `toBeDefined()`

Reply:
No, `not.toBe(null)` is correct according to our coding guidelines. We wanted to avoid `toBeDefined()`.